src/core/logger.ts

```diff
@@ -8,14 +8,25 @@ export interface Logger {
 
 type LogFunction = (...args: any[]) => void
 
-// FUNCTIONS
+export const getDefaultLogger = (): Logger => {
+  if (process.env.NODE_ENV === 'test') {
+    return {
+      log: jest.fn(),
+      warn: jest.fn(),
+      error: jest.fn(),
+    }
+  }
+  if (process.env.NODE_ENV === 'production') {
+    return {
+      log: noop,
+      warn: noop,
+      error: noop,
+    }
+  }
 
-let logger: Logger = console
-
-export function getLogger(): Logger {
-  return logger
-}
-
-export function setLogger(newLogger: Logger) {
-  logger = newLogger
+  return {
+    log: console.log,
+    warn: console.warn,
+    error: console.error,
+  }
```

Comment:
```suggestion
  return console
```

I return this object instead of just `console` because I mimicked `logger.native.ts` assuming there was a problem with react-native builds if not done so, but now I understand why there was a `logger.native.ts` so I can probably change this.